packit/upstream.py

```diff
@@ -64,6 +70,99 @@ def checkout_release(self, version: str) -> None:
         except Exception as ex:
             raise PackitException(f"Cannot checkout release tag: {ex}.")
 
+    def get_commits_to_upstream(
+        self, upstream: str, add_usptream_head_commit=False
+    ) -> List[git.Commit]:
+        """
+        Return the list of commits from current branch to upstream rev/tag.
```

Comment:
Could this be more descriptive please? Is this meant to be a different between current HEAD and the specific ref?

Reply:
> Is this meant to be a different between current HEAD and the specific ref?

Yes and no...;-) It's hard to explain.

- Yes - there are commits between `HEAD` and `ref`.
- No - not all of them. With `first_parent=True` we are choosing commits in one line/path and always choose the first parent. (Commits in the PR and merge commits on `master`.)

I'll try to explain that better.

Reply:
I've changed it a little.